### PR TITLE
implement bio page

### PR DESCRIPTION
**Pull request by masskoder**

### CODE REVIEW

**Comment by rebeccabilbro** [comment]
Thanks for removing those files from the `public` folder @masskoder! It looks like there might be one more file, `public/.DS_Store` that still needs to be deleted. Also, can you please add the `public` folder to our `.gitignore` file at the top level of the repo to make sure we don't accidentally add `public` files to the repo in the future?

One more thing I am seeing is that in the social icons, it is too difficult to see the icon because of a low contrast between the internal icon and the external blue box:
Screen Shot 2022-11-04 at 1 47 11 PM

File: content/en/bio/edwin-schmierer.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+name: "Edwin Schmierer"
+slug: "edwin-schmierer"
+profile: img/team/edwin-schmierer.png
+designation : Founder/COO
+field: The Business
+social :
+  - name : "linkedin"
+    link: 'https://www.linkedin.com/in/ESchmierer/'
```
Comment (rebeccabilbro) [inline content/en/bio/edwin-schmierer.md:9]: Oops, looks like this link is broken -- I think it should be 'https://www.linkedin.com/in/edwinschmierer/'

File: content/en/bio/cletus-razakou.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+name: "Cletus Razakou"
+slug: "cletus-razakou"
+profile: img/team/cletus-razakou.png
+designation : Senior Frontend Engineer
```
Comment (daniellemaxwell) [inline content/en/bio/cletus-razakou.md:5]: Should it be Front-end here to be consistent with line 17?

File: content/en/bio/cletus-razakou.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+  - name : "linkedin"
+    link: 'https://www.linkedin.com/in/cletusrazakou/'
+    icon: 'fa-linkedin-in'
+  - name : "twitter" # themify icon pack : https://themify.me/themify-icons
+    link : "https://twitter.com/masskoder"
+    icon: 'fa-twitter'
+  - name : "github"
+    link: 'https://github.com/masskoder'
+    icon: 'fa-github'
+bio : "Cletus Razakou is a senior front-end engineer at Rotational Labs. He has more than a decade of experience in full-stack development, most often using typescript, ReactJS, Angular, Dart, C# and NodeJS. He also has experience in leading engineering teams building scalable products. Aside from work, he loves spending time with his daughter, reading, traveling to new places, and discovering new foods."
```
Comment (daniellemaxwell) [inline content/en/bio/cletus-razakou.md:17]:
```suggestion
bio : "Cletus Razakou is a senior front-end engineer at Rotational Labs. He has more than a decade of experience in full-stack development, most often using TypeScript, ReactJS, Angular, Dart, C# and NodeJS. He also has experience in leading engineering teams building scalable products. Aside from work, he loves spending time with his daughter, reading, traveling to new places, and discovering new foods."
```

File: content/en/bio/elysee-bleu.md [hunk 4]
```diff
@@ -0,0 +1,15 @@
+---
+name: "Elysee Bleu"
+slug: "elysee-bleu"
+profile: img/team/elysee-bleu.png
+designation : Frontend Engineer
```
Comment (daniellemaxwell) [inline content/en/bio/elysee-bleu.md:5]: Should it be Front-end here to be consistent with line 14?

File: content/en/bio/elysee-bleu.md [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+field: React.JS Specialist
+social :
+  - name : "linkedin"
+    link: 'https://www.linkedin.com/in/elys%C3%A9e-bleu-00078a172/'
+    icon: 'fa-linkedin-in'
+  - name : "github"
+    link: 'https://github.com/elysee15'
+    icon: 'fa-github'
+bio : "Elysee Bleu is a Junior front-end developer with over 2 years of experiences. He also has some knowledges and experiences in back-end development. As a front-end developer, his job is to create user interfaces and maintain the client side web applications. In his spare time, he loves reading, watch movies and take time with his family and friends."
```
Comment (daniellemaxwell) [inline content/en/bio/elysee-bleu.md:14]:
```suggestion
bio : "Elysee Bleu is a Junior front-end developer with over 2 years of experience. He also has some knowledge and experience in back-end development. As a front-end developer, his job is to create user interfaces and maintain the client side web applications. In his spare time, he loves reading, watching movies and spending time with his family and friends."
```

File: content/en/bio/cletus-razakou.md (hunk 2 above)
Comment (daniellemaxwell) [inline content/en/bio/cletus-razakou.md:5]: Should it be Front-end here to be consistent with line 17?

**Review by daniellemaxwell — APPROVED** [review]
This looks great! I had a few small suggestions.

**Review by rebeccabilbro — APPROVED** [review]
Looks great @masskoder! I have added the missing bios and made a few other tiny edits, so I think this should be ready to merge. 

Two requests for follow-on stories to add to Shortcut:
- make the bio text markdown-friendly (so that we can put in links)
- make the bio images a standard size and dimensions (currently they are all different)

**Pull request merged**